build.gradle

```diff
@@ -62,8 +62,8 @@ dependencies {
         build(['name':'github.com/nicksnyder/go-i18n/i18n/bundle', 'version':'991e81cc94f6c54209edb3192cb98e3995ad71c1', 'transitive':false])
         build(['name':'github.com/nicksnyder/go-i18n/i18n/language', 'version':'991e81cc94f6c54209edb3192cb98e3995ad71c1', 'transitive':false])
         build(['name':'github.com/nicksnyder/go-i18n/i18n/translation', 'version':'991e81cc94f6c54209edb3192cb98e3995ad71c1', 'transitive':false])
-        build(['name':'github.com/apache/incubator-openwhisk-client-go/whisk', 'version':'1759868a61729708a8ab1a6de1b409d6e2aea00a', 'transitive':false])
-        build(['name':'github.com/apache/incubator-openwhisk-client-go/wski18n', 'version':'1759868a61729708a8ab1a6de1b409d6e2aea00a', 'transitive':false])
+        build(['name':'github.com/apache/incubator-openwhisk-client-go/whisk', 'version':'defaf9776014c3d9edbffb2671ed166855d0ee60', 'transitive':false])
```

Comment:
assume we need some new go client function and forced a drag on i18n.  However, our functional code did not change to add any new features?

Reply:
ahh, so this is behavior we get from Go client...